### DISCUSSION TITLE
Write and save measured round- and lap times into a file

### DISCUSSION THREAD

**Discussion by Volker-F3F** [gerhardf1/F3F-Timer#6] [Ideas]
Dear Gerhard,
Follows my second feature request ... :-)

After practice sessions I analyze the log files of the radio and your App will give me a further support if for each round there will be a TXT file with following data:
- Date (yyyy.mm.dd) & time (hh.mm.ss) of the file
- Number of laps for this round
- Total time of the round
- Time of each lap

The saved filename should have a prefix related to your App followed by "yyyy.mm.dd-hh.mm"

What do you think?
Sending kind regards from Graz
Volker

**Comment by gerhardf1** [comment]
Hello Volker,

also a good idea, but I've to check first how file writing can be done with MIT App Inventor.

regards
Gerhard

**Comment by gerhardf1** [comment]
I released V.012 where the results are now written to a file.

regards
Gerhard
